.github/workflows/snyk.yaml

```diff
@@ -0,0 +1,36 @@
+name: Snyk test Docker image
+on: 
+  push:
+    branches: [master]
```

Comment:
Maybe more a question to the more experienced developers: Should we also monitor the `radix` branch or inside of Radix good enough? I know that Radix also runs Snyk on the deployed image, but will they inform us if there is a vulnerability?

Reply:
On Radix as far as I see it only shows a "bug" in UI near old and buggy deployment. We don't get any additional notifications as far as I see.

If our security champions think we want a faster reaction to this - then it is good idea to monitor at least Radix branch (test could stay whatever?)